src/createNumbersGenerator.js

```diff
@@ -26,7 +26,27 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  const arr = numbers;
+  let index = 0;
+
+  return () => {
+    if (!arr) {
+      return undefined;
+    }
+
+    let num;
+
+    if (min && max) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max